hrishikeshpaul/vibey
# Get and add rooms

This PR adds the functionality to get the current rooms along with adding a user (that isn't the host) to the room.

### How to test

1. create a room
2. check your redis cli running on 6380. The room should be added. there should be one user (host).
3. In incognito, try to log in from a friend's spotify account. currently multiple users cannot enter the same room (obv). Once you do that, you'll see the room list being populated on the home page. Click on the room you just reacted. 
4. From the host, start a playlist. It should start playing on both the accounts.
5. Once you check your redis cli again, you'll see that stuff in that object is updated.

I'm not sure what bugs can arrive, but this is def something you need to look at pretty closely. Try to spend some time on the PR.

**Note** Ignore the code in `core/player`. Thats all the code from the NPM package with renamed files.
